src/rustdoc/membership.md

```diff
@@ -0,0 +1,116 @@
+
+  Tip: some baseline rules around bors permissions are: don't do a `try` build unless you have
+  done a check for malicious code first and don't `r+` unless you are reasonably confident that
+  you can effectively review the code in question.
+- Rustdoc team members are members of the Rust organization so they can modify labels and be
+  assigned to issues.
+- Members become a part of the `rust-lang/rustdoc` team on GitHub, so that they receive pings
+  when people are looking to address the team as a whole.
+- Members are listed on the [rust-lang.org web page].
+
+It also implies some obligations (in some cases, optional obligations):
+
+- Members may take part in various other [maintainer activities] to help the team.
+- Members are held to a higher standard than ordinary folk when it comes to the [Code of
+  Conduct][CoC].
+
+[rust-lang.org web page]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
+
+## What it means to be a rustdoc member
+Once you're a member of the rustdoc team, a number of events will happen:
+
+- You will gain access to a private Zulip stream, where internal discussions happen.
+- You will also be subscribed to the `all@rust-lang.org` mailing list. See
+  [this file](https://github.com/rust-lang/team/blob/HEAD/teams/all.toml) to check how subscriptions
+  to mailing lists work. It's a very low-volume mailing list (maybe a few emails per year), it's a
+  way to communicate things to all contributors. We will not send you spam from this address.
+
+## Roles in the rustdoc team
+
+Rustdoc has multiple different areas and team members are not working on all of them. Currently
+we have three main areas:
+
+- Front-end: Everything related to HTML/CSS/JS
+- JSON backend: Work on the `--output-format=json` backend.
+- Internals: The internals of rustdoc: interacting with the compiler, doctests, generating
+  rustdoc internal code representation, parsing command line arguments, lints, etc.
+
+These groups are NOT full-fledge teams, and as such, to be part of any of these groups, you need to
+be a member of the rustdoc team.
+
+For now, only the front-end group has an official status in the
+[team repository](https://github.com/rust-lang/team) and is called `rustdoc-frontend`. If a rustdoc
+team member is interested to be part of this group, they can ask to be added into it.
+
+Being part of the front-end group doesn't change your rustdoc team membership. However
+you will be assigned for reviews on front-end pull requests and on front-end FCPs.
+
+## How promotion decisions are made
+[hdam]: #how-promotion-decisions-are-made
+
+After an individual has been contributing to rustdoc for a while, they may be nominated in the
+private Zulip rustdoc team channel by an existing team member.
+
+The rustdoc team members will check to see if there are concerns with extending a membership
+invitation to the individual and after a week (barring no objections), an invitation will be
```

Comment:
Kind of a nitpick, but should we do 10 days to match FCP length?

Reply:
No opinion. I'll update to 10 days. As usual, can be revisited if needed later on.